src/inmanta/data/dataview.py

```diff
@@ -926,20 +926,11 @@ def get_base_query(self) -> SimpleQueryBuilder:
         return query_builder
 
     def construct_dtos(self, records: Sequence[Record]) -> Sequence[ResourceHistory]:
-        def get_attributes(record: Record) -> JsonType:
-            attributes = record["attributes"]
-            if "version" not in attributes:
-                # Due to a bug, the version field has always been present in the attributes dictionary.
-                # This bug has been fixed in the database. For backwards compatibility reason we here make sure that the
-                # version field is present in the attributes dictionary served out via the API.
-                attributes["version"] = record["model"]
-            return attributes
-
         return [
             ResourceHistory(
                 resource_id=self.rid,
                 attribute_hash=record["attribute_hash"],
-                attributes=get_attributes(record),
+                attributes=cast(dict[str, Any], record["attributes"]),
```

Comment:
Doesn't really hurt here, so not a strict change request, but I don't think we should ever cast to `Any`, unless we really want to disable type checking. `object` could be appropriate. But honestly I find it even more appropriate to simply accept this error in the mypy baseline.